scripts/trlite

```diff
@@ -334,17 +333,17 @@ if [ "$RUN" == "all" -o "$RUN" == "2" ]; then
     SENSORS=(Accelerometer)
     write_modules_test $TMPFILE $MODULES $SENSORS
 
-    # also, try printing a float w/o PRINT_FLOAT
+    # also, try printing a float
     echo "console.log(3.14159);" >> $TMPFILE
     try_command "k64f module" make $VERBOSE JS=$TMPFILE ROM=256 BOARD=frdm_k64f
 
     MODULES=(net ws)
     write_modules_test $TMPFILE $MODULES $SENSORS
 
-    # also, try printing a float w/ PRINT_FLOAT
+    # also, try printing a float
     echo "console.log(3.14159);" >> $TMPFILE
 
-    try_command "k64f net" make $VERBOSE JS=$TMPFILE PRINT_FLOAT=on ROM=256 BOARD=frdm_k64f
+    try_command "k64f net" make $VERBOSE JS=$TMPFILE ROM=256 BOARD=frdm_k64f
```

Comment:
The point of these two sets of alternatives was to test both w/ and without. There's no point in that anymore so we could at least remove two of these... and maybe all of them. (We don't really have a ZJS feature to test anymore per se.)